cmd/monaco/download/downloaders.go

```diff
@@ -33,28 +36,92 @@ func makeDownloaders(options downloadConfigsOptions) (downloaders, error) {
 		return nil, err
 	}
 
-	var automationDownloader download.Downloader[v2.AutomationType] = dlautomation.NoopAutomationDownloader{}
+	var automationDownloader download.Downloader[config.AutomationType] = dlautomation.NoopAutomationDownloader{}
 	if clients.Automation() != nil {
 		automationDownloader = dlautomation.NewDownloader(clients.Automation())
 	}
-	var settingsDownloader download.Downloader[v2.SettingsType] = settings.NewDownloader(clients.Settings())
-	var classicDownloader download.Downloader[v2.ClassicApiType] = classic.NewDownloader(clients.Classic())
-	return downloaders{settingsDownloader, classicDownloader, automationDownloader}, nil
+	var settingsDownloader download.Downloader[config.SettingsType] = settings.NewDownloader(clients.Settings())
+	var classicDow download.Downloader[config.ClassicApiType]
+	if d, e := classicDownloader(clients.Classic(), options); e != nil {
```

Comment:
```suggestion
	if d, e := classicDownloader(clients.Classic(), options); e == nil {
```

this logic is inverted, currently if no error happens, a nil downloader and nil error are returned (else case).

I've run into this while trying it locally after commenting out the non-compiling tests. 
Please ensure this logic is properly covered by tests (if it's covered once the existing tests are fixed great, else please add test cases for the downloader initialization) !